apps/src/templates/instructions/InstructionsCsfRightCol.jsx

```diff
@@ -82,11 +81,7 @@ class InstructionsCsfRightCol extends React.Component {
   }
 
   getColumnHeight() {
-    const collapseButtonHeight = getOuterHeight(this.collapser, true);
-    const scrollButtonsHeight = this.scrollButtons
-      ? this.scrollButtons.getMinHeight()
-      : 0;
-    return collapseButtonHeight + scrollButtonsHeight;
```

Comment:
This didn't seem to be having the desired effect, and was only measuring the height of the collapse button, because the scroll buttons' parent had a height of 0.

Reply:
maybe add a comment around why this always returns 0?